src/main/java/braintree/BazelDeps.java

```diff
@@ -75,9 +76,10 @@ private void printWorkspace(Map<Artifact, Set<Artifact>> dependencies,
     if(repository != null){
       System.out.format("maven_server(name = \"default\", url = \"%s\")\n", repository);
     }
-    dependencies.values().stream()
-      .flatMap(Collection::stream)
-      .filter(artifact -> !excludeDependencies.contains(artifact))
+    Set<Artifact> uniqueDependencies = dependencies.values().stream()
+      .flatMap(Collection::stream).collect(Collectors.toSet());
```

Comment:
Super minor, but do you mind putting the collect on the following line?

Reply:
Sure, updated.

Reply:
Thanks!